src/graphics/mod.rs

```diff
@@ -99,7 +102,4 @@ pub mod layers {
 	pub const CYCLE_CENTER_SPRITES: f32 = -300.0;
 	pub const CYCLE_LINKS: f32 = -400.0;
 	pub const BACKGROUND: f32 = -550.0;
```

Comment:
I think these are starting to need doc comments with units and stuff.

Reply:
I don't think these *have* any meaningful units. Z depths are really only meaningful when you can compare them to others.